lib/ex_doc/formatter/html.ex

```diff
@@ -15,10 +15,21 @@ defmodule ExDoc.Formatter.HTML do
   def run(module_nodes, config) when is_map(config) do
     config = normalize_config(config)
     output = Path.expand(config.output)
-    File.rm_rf! output
-    :ok = File.mkdir_p output
+    build = Path.join(output, ".build")
+
+    if File.exists? build do
+      File.stream!(build)
+      |> Enum.map(fn(line) ->
+        line = String.trim(line)
+        File.rm!(Path.join(output, line))
```

Comment:
I think `File.rm` here and on the lines below would be enough. If someone removed the file, we don't want the build process to fail forever until the user deletes the whole directory.